tests/lax_numpy_test.py

```diff
@@ -160,6 +160,15 @@ def _shapes_are_equal_length(shapes):
   return all(len(shape) == len(shapes[0]) for shape in shapes[1:])
 
 
+def _broadcastable_to(shape_1, shape_2):
+  try:
```

Comment:
Could we simplify this to `return jnp.broadcast_shapes(shape_1, shape_2) == shape_2`?